File: TTS/app/src/main/java/com/example/tts/MainActivity.java
package com.example.tts;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.speech.tts.TextToSpeech;
import android.widget.Toast;

import java.util.Locale;

public class MainActivity extends AppCompatActivity {


    TextToSpeech tts;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tts = new TextToSpeech(this, new TextToSpeech.OnInitListener() {
            @Override
            public void onInit(int i) {
                tts.setLanguage(new Locale("pt-BR"));
                Toast.makeText(
                        MainActivity.this,
                        "TTS Iniciado com sucesso!",
                         Toast.LENGTH_SHORT).show();

                tts.speak(
                        "Exemplo de fala pelo android",
                         TextToSpeech.QUEUE_FLUSH,
                        null);
            }
        });
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        if (tts != null) {
            tts.shutdown();
        }
    }
}

File: ExemploDeKotlin/app/src/main/java/com/example/exemplodekotlin/MainActivity.kt
package com.example.exemplodekotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

fun Int.somar(b : Int) {this + b}

class MainActivity : AppCompatActivity() {

    //val é constante e var é variável

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val contatoJava = ContatoJava(
            1,
            "Ricardo",
            "ricardorodrigo.oliveira@gmail.com",
            "11969051743")

        val contatoKotlin = ContatoKotlin(
            1,
            "Ricardo",
            null,
            null)

        txtExemplo.text = "Qualquer coisa aqui"

        contatoJava.nome = "Ricardo"
        contatoJava.email = "ricardorodrigo.oliveira@gmail.com"

        var soma:String = 10.somar(5) as String
        Toast.makeText(this, soma, Toast.LENGTH_SHORT).show()

    }

    fun somar(a : Int, b : Int) {a + b}
}
